rules/cre-2025-0104/istio-ambient-xds.yaml

```diff
@@ -0,0 +1,48 @@
+rules:
+  - metadata:
+      kind: prequel
+      id: bR7xUvNyLdGQ2pTjWzYkEp
```

Comment:
Can you please change the ID? This is the same as https://github.com/prequel-dev/cre/pull/90/files and conflicts.